app/src/main/res/values/strings.xml

```diff
@@ -171,91 +171,92 @@
-    <string name="crash_report_comment_prompt">You can add a comment below:</string>
+    <string name="crash_report_text">EVMap crashed. Please send a crash report to the developer.</string>
+    <string name="crash_report_comment_prompt">Comment below:</string>
     <string name="powered_by_mapbox">powered by Mapbox</string>
-    <string name="pref_search_provider">Place search provider</string>
-    <string name="pref_search_provider_info"><![CDATA[Data for place search, especially from Google Maps, is relatively expensive. If you use this feature often, please consider making a donation through \"About EVMap -> Donate\".]]></string>
+    <string name="pref_search_provider">Geo-searches</string>
+    <string name="pref_search_provider_info"><![CDATA[Data for geo-searches. Expensive to fetch, especially from Google Maps. Please consider donating through “About” → “Donate”.]]></string>
     <string name="github_sponsors">GitHub Sponsors</string>
-    <string name="donate_desc">Support EVMap\'s development with a one-time donation</string>
-    <string name="github_sponsors_desc">Support EVMap on GitHub Sponsors</string>
+    <string name="donate_desc">Make a one-time donation to support EVMap development</string>
+    <string name="github_sponsors_desc">Support EVMap</string>
     <string name="unnamed_filter_profile">Unnamed filter profile</string>
     <string name="privacy_link">https://evmap.vonforst.net/en/privacy.html</string>
     <string name="faq_link">https://evmap.vonforst.net/en/faq.html</string>
     <string name="chargeprice_faq_link">https://evmap.vonforst.net/en/chargeprice_faq.html</string>
     <string name="required">required</string>
     <string name="edit_filter_profile">Edit “%s”</string>
-    <string name="pref_search_delete_recent">Delete recent search results</string>
-    <string name="deleted_recent_search_results">Recent search results have been deleted</string>
-    <string name="settings_data_sources">Data sources</string>
+    <string name="pref_search_delete_recent">Delete recent searches</string>
+    <string name="deleted_recent_search_results">Recent searches deleted</string>
+    <string name="settings_data_sources">Sources</string>
     <string name="help">Help</string>
     <string name="settings_android_auto">Android Auto</string>
-    <string name="pref_chargeprice_allow_unbalanced_load">Enable unbalanced load</string>
-    <string name="pref_chargeprice_allow_unbalanced_load_summary"><![CDATA[Allow charging with >4.5 kW at AC stations for cars with single-phase charger]]></string>
-    <string name="pref_map_rotate_gestures_enabled">Enable map rotation</string>
-    <string name="pref_map_rotate_gestures_on">Map can be rotated with two-finger gesture</string>
-    <string name="pref_map_rotate_gestures_off">Map will be fixed to north-up</string>
+    <string name="pref_chargeprice_allow_unbalanced_load">Unbalanced load</string>
```

Comment:
```suggestion
    <string name="pref_chargeprice_allow_unbalanced_load">High load</string>
```
With your [description below](https://github.com/johan12345/EVMap/pull/218/files#r952862904), unless it is a concern for power correction/inductive/reactive load, I don't understand why it isn't just "High load", as opposed to "unbalanced"

Reply:
Well, it does have to do with three-phase electricity grids and not having a high load on a single phase of a three-phase grid. In Germany, Austria and Switzerland, such a rule (limit to 20 A or even 16 A for single-phase loads) applies at least to all residential chargers, for public chargers it depends on the operator if it's actually enforced. It shouldn't be a concern in countries that have just a single-phase grid.